src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2217,7 +2217,35 @@ private static string[] ParseArgv(ProcessStartInfo psi)
         {
             var argvList = new List<string>();
             argvList.Add(psi.FileName);
-            argvList.AddRange(psi.Arguments.Split(' '));
+
+            var argsToParse = psi.Arguments.Trim();
+            var argsLength = argsToParse.Length;
+            for (int i=0; i<argsLength; )
+            {
+                var iStart = i;
+         
+                switch (argsToParse[i])
+                {
+                    case '"':
+                        // Special case for arguments within quotes
+                        // Just return argument value within the quotes
+                        while ((++i < argsLength) && argsToParse[i] != '"') { };
```

Comment:
My initial concern regarded unmatched double quotes. Thinking about it some more, two other things came to mind:
1. Strings that use single quotes
2. Argument lists that include other commands. Do we do validation on it? I'm wondering about something like this: `arg1 arg2; rm -rf *`